hack/make/.build-rpm/docker-engine.spec

```diff
@@ -50,6 +50,27 @@ Requires: kernel-uek >= 3.8
 Requires: device-mapper >= 1.02.90-2
 %endif
 
+# docker-selinux conditional
+%if 0%{?fedora} >= 22 || 0%{?centos} >= 7 || 0%{?rhel} >= 7 || 0%{?oracle} >= 7 
+%global with_selinux 1
+%endif
+
+# start if with_selinux
+%if 0%{?with_selinux}
+# Version of SELinux we were using
+%if 0%{?fedora} >= 22
+%global selinux_policyver 3.13.1-119
+%else
+%global selinux_policyver 3.13.1-39
```

Comment:
This is too high for EL7-derived distros. The latest RPM in EL7 is `selinux-policy-3.13.1-23.0.1.el7_1.13.noarch`